### DISCUSSION TITLE
Unable to install OKD 4.8 on vSphere UPI (read udp [::1]:50918->[::1]:53: read: connection refused)

### DISCUSSION THREAD

**Discussion by Wuk-jvi** [okd-project/okd#942] [Q&A]
Hi,

I'm trying to install OKD 4.8 on vSphere UPI, same  way as I did previous clusters, but now I have issues that I cannot fix.
After installing bootstrap node, I tried to install first master node, but after reboot I get following messages:
`ignition[674]: GET error: Get https://api-int.test48.venera.internal:22623/config/master: dial tcp: lookup api-int.test48.venera.internal on [::1]53: read udp [::1]:50918->[::1]:53: read: connection refused`
I tried to install using kernel arguments, I also tried to install using coreos-installer from command prompt after boot from ISO but results are same, tried also iso, and raw.xz images.

ISO: fedora-coreos-34.20210919.3.0-live.x86_64.iso
RAW.XZ: fedora-coreos-34.20210919.3.0-metal.x86_64.raw.xz

kernel parameter install:
`ip=10.10.20.51::10.10.20.1:255.255.255.0:test48-m1:ens192:none nameserver=10.10.10.11 nameserver=10.10.10.12 nameserver=10.10.10.13 coreos.inst.install_dev=/dev/sda coreos.inst.image_url=http://www-okd.venera.internal/4.8/fcos.raw.xz coreos.inst.ignition_url=http://www-okd.venera.internal/4.8/master.ign`

coreos-installer:
`sudo nmtui` - set same ip configuration as for kernel parameter install
`curl http://www-okd.venera.internal/4.8/master.ign -o master.ign`
`sudo coreos-installer install --copy-network --ignition-file=master.ign --firstboot-args 'rd.neednet=1' /dev/sda` - first tried without `--firstboot-args
`

I'm stuck, and at this moment I have no idea what to try next.

**Comment by vrutkovs** [comment]
>ISO: fedora-coreos-34.20210919.3.0-live.x86_64.iso

See https://github.com/openshift/okd/issues/783
